src/components/home/devRevenueCosts/Options.js

```diff
@@ -0,0 +1,74 @@
+import { RevenueCostsA, RevenueCostsM } from './Data';
+
+export const RevenueCostsActual = {
+  data: RevenueCostsA,
+  isGroup: true,
+  seriesField: 'name',
+  xField: 'projectName',
+  yField: 'value',
+  yAxis: {
+    min: 0,
+    max: 600000,
+  },
+  minColumnWidth: 20,
+  maxColumnWidth: 20,
+  dodgePadding: 6,
+  color: ({ name }) => {
+    return name === 'Grand Total Costs' ? '#FF9F5A' : '#7BB99F';
+  },
+  legend: {
+    position: 'top-right',
+    layout: 'horizontal',
+    marker: {
+      symbol: 'circle',
+      r: 5,
+      lineWidth: 2,
+      strokeColor: ({ color }) => color, 
+      fill: 'transparent',
+    },
+    },
+};
+
+export const RevenueCostsMonth = {
+  data: RevenueCostsM.flat(),
+  isGroup: true,
+  seriesField: 'type',
+  xField: 'type', 
+  yField: 'value',
+  groupField: 'name',
+  yAxis: {
+    min: 0,
+    max: 260000,
+  },
+  minColumnWidth: 20,
+  maxColumnWidth: 20,
+  dodgePadding: 6,
+  color: ({ type }) => {
+    if (type === 'Grand Total Planned Revenue') {
```

Comment:
this is good for hardcoded data. But if we change something this will not work at all.